src/20241229_AaveV3EthereumLido_Deploy10MGHOIntoAaveV3LidoInstance/Deploy10MGHOIntoAaveV3LidoInstance.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Deploy 10M GHO into Aave v3 Lido Instance"
+author: "karpatkey_TokenLogic & BGD Labs"
+discussions: "https://governance.aave.com/t/arfc-mint-deploy-10m-gho-into-aave-v3-lido-instance/19700"
+snapshot: "https://snapshot.box/#/s:aave.eth/proposal/0x4af37d6b4a1c9029c7693c4bdfb646931a9815c4a56d9066ac1fbdbef7cd15e5"
+---
+
+## Simple Summary
+
+Following the addition of GHO to the Lido instance of Aave v3 on Ethereum, this publication proposes supporting GHO liquidity by minting and depositing 10M units of GHO and depositing into Aave Lido.
+
+## Motivation
+
+By providing liquidity on the Lido instance, the Aave DAO shall provide the initial bootstrapping liquidity in a very cost efficient manner and in doing so, enahnce the DAO's revenue generated from the Lido instance. GHO holders will be able to deposit GHO to earn the deposit yield whilst benefit from small than otherwise concentration risk within the reserve due to the DAO providing the bootstrapping liquidity.
```

Comment:
```suggestion
By providing liquidity on the Lido instance, the Aave DAO shall provide the initial bootstrapping liquidity in a very cost efficient manner and, in doing so, enhance the DAO's revenue generated from the Lido instance. GHO holders will be able to deposit GHO to earn the deposit yield whilst benefit from small than otherwise concentration risk within the reserve due to the DAO providing the bootstrapping liquidity.
```